cypress/e2e/login.cy.js

```diff
@@ -0,0 +1,16 @@
+describe(`Login Test`, () => {
+  it(`should log in with valid credentials`, () => {
+    cy.visit(`https://teehbee.github.io/social-media-client-ca/`);
```

Comment:
```suggestion
    cy.visit(`/`);
```

Cypress runs its own server for testing. By targeting the remote version, you are not testing the code in each pull request - you are testing completed code that has already been reviewed and merged!